_posts/2020-01-19-zarzadzanie-zadaniami.md

```diff
@@ -0,0 +1,288 @@
+
+Przeglądanie zadań jest proste. Zadania bez wskazanego projektu wpadają do skrzynki odbiorczej. Jest to worek na wszystko co wpadło (o ile podczas tworzenia zadania nie podałem wprost projektu). Przeglądając skrzynkę odbiorczą mogę:
+* uzupełnić opis zadania
+  * notatki w formie tekstu i załączników
+* dodać podzadania
+* ustawić priorytet
+  * 5 poziomów
+  * steruje kolejnością wyświetlania zadań (prosto, bez ręcznego sortowania i nadmiernego wyboru)
+* przypisać do projektu
+  * pozwala zdefiniować osobne worki zadań
+  * ułatwia _"ogarnięcie"_ dużej ilości zadań
+* ustawić etykietę
+  * wykorzystuję jako wskazówka czego potrzebuję do wykonania zadania
+  * jedno zadanie może mieć wiele etykiet
+  * przykłady moich etykiet: email, wiki, jira, biuro, cisza
+  * bardzo przydatne przy wykorzystaniu filtrów
+* ustawić termin
+* ustawić przypomnienie
+  * czasowe
+  * geograficzne (np. _"jak wejdę do Castorama"_) - brzmi fajnie, przydatność taka sobie w moim przypadku
+
+Wybór zadania do wykonania jest prosty. Opieram się o filtry wybierające zadania na dany moment. Dla przykładu:
+* filtr "K9" (od nazwy naszego biura):
+  * etykiety: biuro lub jira lub wiki lub email lub telefon lub "nie cisza"
+  * termin: dzisiaj lub brak terminu (jeżeli na jutro to dzisiaj nie wrzucaj mi tego na ekran)
+* filtr "pociąg":
+  * etykiety: jira lub wiki lub email lub cisza
+  * termin: dzisiaj lub brak terminu
+* filtr "wieczór":
+  * etykiety: jira lub wiki lub email lub cisza lub "nie biuro"
+
+Zadania widoku filtra są posortowane po priorytetach. Najważniejsze na górze. Aplikacja Chrome dodana do autostartu zapewnia że lista zadań będzie zawsze widoczna na pulpicie komputera.
+
+## Kiedy to wszystko robić?
+<!-- ciagle ;) ale postaram się napisać dlaczego -->
+
+Dodajemy zadania ciągle, na bieżąco. Nie chcemy pamiętać o dodaniu zadania.
+
+Przegląd zadań - regularnie. U mnie sprawdza się metoda _"pozytywnego skojarzenia"_: poranny przegląd zadań zawsze przy **porannej kawce**. Należy pamiętać, że przegląd można robić też w miarę potrzeb nawet jeśli niedawno zrobiło się planowany. Jeżeli np. zrealizowaliśmy już wszystkie zadania na dzisiaj (rzadko) albo sprawy bieżące przygniotły nas bardziej niż zakładaliśmy (często).
+
+Wykonywanie zadań - ciągle. Jeżeli siadam przy biurku to nie zastanawiam się co tu by zrobić. Patrzę na listę w odpowiednim filtrze i jadę od góry. Zadanie możemy również wykonywać w odpowiedzi na przypomnienie. Polecam jednak ostrożność w tym aspekcie. Przypomnienia potrafią skutecznie wybić z rytmu.
+
+
+### Kiedy robię coś nie tak?
+
+Po czym mogę poznać, że coś robię nie tak? Mam skonfigurowane narzędzie. Jestem zmotywowany. Coś tam wychodzi, ale czy na pewno? Poniższe objawy powinny nas sprowokować do optymalizacji lub zmiany podejścia:
+
+* nie zapisuję zadań
+  * nie mam przy sobie narzędzia
+  * nie wiem na kiedy to
+  * nie wiem czego będę potrzebował
+  * nie wiem czy warto
```

Comment:
3x przecinki
  * nie wiem, na kiedy to
  * nie wiem, czego będę potrzebował
  * nie wiem, czy warto